bld/namelist_files/use_cases/20thC_transient.xml

```diff
@@ -26,6 +26,14 @@
 <stream_year_last_ndep  phys="clm5_0" use_cn=".true."   >2005</stream_year_last_ndep>
 <model_year_align_ndep  phys="clm5_0" use_cn=".true."   >1850</model_year_align_ndep>
 
+<stream_year_first_fan phys="clm4_5" use_cn=".true." ndepsrc="stream" >2010</stream_year_first_fan>
+<stream_year_last_fan  phys="clm4_5" use_cn=".true." ndepsrc="stream" >2010</stream_year_last_fan>
+<model_year_align_fan  phys="clm4_5" use_cn=".true."   >2010</model_year_align_fan>
+
+<stream_year_first_fan phys="clm5_0" use_cn=".true." ndepsrc="stream" >2010</stream_year_first_fan>
```

Comment:
bigger question than this pull request, but what's the vision for building a transient dataset to afford historical and future simulations with FAN?

Reply:
You could directly slide a transient dataset in, by just updating the filename and having it run over the correct years. This is one reason for using the streams mechanism so that you can do it very easily.